src/jit/lclvars.cpp

```diff
@@ -2676,6 +2676,173 @@ int __cdecl Compiler::WtdRefCntCmp(const void* op1, const void* op2)
     return 0;
 }
 
+#if defined(_TARGET_AMD64_) && defined(FEATURE_CORECLR)
+
+/* static */
+int __cdecl Compiler::WtdRefCntCmpSimpleGC(const void* op1, const void* op2)
+{
```

Comment:
Decided to ditch this thing, it's just too awkward and ugly (and a maintenance danger); thanks for flagging it.